lib/rpush/client/redis/webpush/notification.rb

```diff
@@ -0,0 +1,15 @@
+module Rpush
+  module Client
+    module Redis
+      module Webpush
+        class Notification < Rpush::Client::Redis::Notification
+          include Rpush::Client::ActiveModel::Webpush::Notification
+
+          def time_to_live=(value)
```

Comment:
How come this isn't required for active_record?

Reply:
same reason as with app.rb